commands/logs_e2e_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/client"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestlogsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Display single log line", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(s.th.SystemAdminClient, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(data, 2)
+		s.Contains(data[1], "info app/plugin.go:223 Syncing plugins from the file store")
+	})
+
+	s.RunForAllClients("Display logs", func(c client.Client) {
```

Comment:
I think we might want to be more definitive with the test case. Probably something like: "Display in logrus for formatting."

Reply:
you mean to change the test name to "Display in logrus for formatting"?
Also updated the test check.